.github/workflows/snap.yaml

```diff
@@ -1,6 +1,8 @@
 name: Snap
 
-on: [push]
+on:
+  push:
+    branches: [ main ]
```

Comment:
@Kethku i was fixing build for m1, when i noticed this. Bulding and publishing snaps on each push.

That's probably not how it supposed to work. I changed this to publish snaps only for main branch.

Reply:
I think you're likely correct. Good catch